Research/ResearchUI/iOS/Step View Controllers/RSDScrollingOverviewStepViewController.swift

```diff
@@ -141,58 +139,279 @@ open class RSDScrollingOverviewStepViewController: RSDOverviewStepViewController
+        let background = self.backgroundColorTile ?? RSDGrayScale().white
+        let contentTile = designSystem.colorRules.tableCellBackground(on: background, isSelected: isSelected)
+
+        contentView.backgroundColor = contentTile.color
+        titleLabel?.textColor = designSystem.colorRules.textColor(on: contentTile, for: RSDTitleHeaderCollectionViewHeader.titleTextType)
+        titleLabel?.font = RSDTitleHeaderCollectionViewHeader.titleLabelFont(for: designSystem)
+    }
+    
+    fileprivate static func titleLabelFont(for designSystem: RSDDesignSystem) -> UIFont {
+        return designSystem.fontRules.font(for: RSDTitleHeaderCollectionViewHeader.titleTextType)
+    }
+
+    override open func setDesignSystem(_ designSystem: RSDDesignSystem, with background: RSDColorTile) {
+        super.setDesignSystem(designSystem, with: background)
+        updateColorsAndFonts()
+    }
+
+    private func commonInit() {
+        // Add the title label.
+        titleLabel = RSDTitleHeaderCollectionViewHeader.createTitleLabel()
+        contentView.addSubview(titleLabel!)
+
+        titleLabel!.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel!.rsd_alignCenterHorizontal(padding: 0.0)
+        titleLabel!.rsd_alignToSuperview([.top], padding: RSDTitleHeaderCollectionViewHeader.kCollectionHeaderTopMargin)
+        titleLabel!.rsd_alignToSuperview([.bottom], padding: RSDTitleHeaderCollectionViewHeader.kCollectionHeaderBottomMargin)
+
+        updateColorsAndFonts()
+        setNeedsUpdateConstraints()
+    }
+    
+    fileprivate static func createTitleLabel() -> UILabel {
+        let label = UILabel()
+        label.numberOfLines = 1
+        label.textAlignment = .center
+        return label
+    }
+}
+
+/// `RSDOverviewCollectionViewCell` shows a vertically stacked image icon and title label.
+@IBDesignable open class RSDOverviewCollectionViewCell: RSDCollectionViewCell {
+
+    let kCollectionCellVeritcalItemSpacing = CGFloat(6)
+    
+    @IBOutlet public var titleLabel: UILabel?
+    @IBOutlet public var imageView: UIImageView?
+
+    open private(set) var titleTextType: RSDDesignSystem.TextType = .small
+
+    public override init(frame: CGRect) {
+        super.init(frame:frame)
+        commonInit()
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder:aDecoder)
+        commonInit()
+    }
+
+    func updateColorsAndFonts() {
+        let designSystem = self.designSystem ?? RSDDesignSystem()
+        let background = self.backgroundColorTile ?? RSDGrayScale().white
+        let contentTile = designSystem.colorRules.tableCellBackground(on: background, isSelected: isSelected)
+
+        contentView.backgroundColor = contentTile.color
+        titleLabel?.textColor = designSystem.colorRules.textColor(on: contentTile, for: titleTextType)
+        titleLabel?.font = designSystem.fontRules.font(for: titleTextType, compatibleWith: traitCollection)
```

Comment:
See comment above. Alternatively, you could use `baseFont()` here instead.

Reply:
Same comment as above